.github/workflows/publish.yml

```diff
@@ -0,0 +1,52 @@
+# This github action builds Alibi Detect and publishes it to PyPI.
+# It is only run when a new tag is pushed to github.
+# See https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+name: Publish 📦 to PyPI 
+
+# Only run when tags starting with "v" are pushed
+on: 
+  push:
+    tags:
+      - 'v*'
+  release:
+    types: [published]
+
+jobs:
+  build-and-publish:
+    name: Build and publish
+    runs-on: ubuntu-18.04
+
+    steps:
+    # Setup Python
+    - uses: actions/checkout@master
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.9
+
+    # Build
+    - name: Build a binary wheel and a source tarball
+      run: make build
+
+    # Publish to Test PyPI if this action was triggered by pushing a v* tag
+    - name: Publish 📦 to Test PyPI
+      if: github.event_name == 'push'  # check for `push` since action only triggered for tag pushes
+      run: |
+        python -m pip install .
+        echo "Dummy release of alibi-detect..."
+        python -c "from alibi_detect.version import __version__; print(__version__)" 
```

Comment:
Doing this temporarily to check that the correct commit is checked out (and thus published). If triggered by a tag being pushed that is checked out. If triggered by a release, the tag linked to that release is checked out (see [here](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release)). The test [here](https://github.com/ascillitoe/alibi-detect/actions/runs/3489129178/jobs/5838829815#step:6:143) confirms this, with `0.10.5alpha2` being printed.